rate counter from vsql is always zero

## Steps to reproduce the problem

1.  Add test to parser package:
```golang

func Test_RateCounter(t *testing.T) {
	require := assertions(t)

	a := require.Build(`APPLICATION app1();
	WORKSPACE w (
		TABLE t INHERITS sys.CDoc();
		RATE r 1 PER HOUR PER WORKSPACE;
	)`)
	w := a.Workspace(appdef.NewQName("pkg", "w"))
	typ := w.Type(appdef.NewQName("pkg", "r"))
	r, ok := typ.(appdef.IRate)
	require.True(ok)
	require.NotNil(r)

	require.EqualValues(1, r.Count(), "error here") // <-- error here

	require.EqualValues(time.Hour, r.Period())
	require.False(r.Scope(appdef.RateScope_AppPartition))
	require.True(r.Scope(appdef.RateScope_Workspace))
	require.False(r.Scope(appdef.RateScope_IP))
	require.False(r.Scope(appdef.RateScope_User))
}
```
2. Run test

<details><summary>Test results</summary>

```console
Running tool: C:\Program Files\Go\bin\go.exe test -timeout 30s -run ^Test_RateCounter$ github.com/voedger/voedger/pkg/parser -count=1 -v

=== RUN   Test_RateCounter
    d:\sigma-auto\voedger\pkg\parser\impl_test.go:2607:
        	Error Trace:	d:/sigma-auto/voedger/pkg/parser/impl_test.go:2607
        	Error:      	Not equal:
        	            	expected: int(1)
        	            	actual  : uint32(0x0)
        	Test:       	Test_RateCounter
        	Messages:   	error here
--- FAIL: Test_RateCounter (0.02s)
FAIL
FAIL	github.com/voedger/voedger/pkg/parser	0.528s
```
</details>

## Expected behavior

Test should be passed.

## Additional context

- [analyzeRate](https://github.com/voedger/voedger/blob/e34e3f83de7c607073afad9e470a71e21a75c528/pkg/parser/impl_analyse.go#L510)

<details><summary>Proposed fix</summary>

```golang
func analyzeRate(r *RateStmt, c *iterateCtx) {

	if r.Value.Count != nil {
		r.Value.count = uint32(*r.Value.Count)
	}
	if r.Value.Variable != nil {
		resolve := func(d *DeclareStmt, p *PackageSchemaAST) error {

			var count int32
			var resolved bool
			if c.variableResolver != nil {
				count, resolved = c.variableResolver.AsInt32(p.NewQName(d.Name))
			}
			if !resolved {
				count = d.DefaultValue
			}
			if count <= 0 {
				return ErrPositiveValueOnly
			}
			r.Value.count = uint32(count)
			return nil
		}
		if err := resolveInCtx(*r.Value.Variable, c, resolve); err != nil {
			c.stmtErr(&r.Value.Variable.Pos, err)
		}
	}
	r.workspace = c.mustCurrentWorkspace()
}

```
</details>

--- a/pkg/parser/impl_analyse.go
+++ b/pkg/parser/impl_analyse.go
@@ -529,6 +529,8 @@ func analyzeRate(r *RateStmt, c *iterateCtx) {
 		if err := resolveInCtx(*r.Value.Variable, c, resolve); err != nil {
 			c.stmtErr(&r.Value.Variable.Pos, err)
 		}
+	} else if r.Value.Count != nil {
+		r.Value.count = *r.Value.Count
 	}
 	r.workspace = c.mustCurrentWorkspace()
 }

--- a/pkg/parser/impl_test.go
+++ b/pkg/parser/impl_test.go
@@ -10,6 +10,7 @@ import (
 	"fmt"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 
@@ -2733,7 +2734,8 @@ func Test_RatesAndLimits(t *testing.T) {
 		typ := w.Type(appdef.NewQName("pkg", "r"))
 		r, ok := typ.(appdef.IRate)
 		require.True(ok)
-		require.NotNil(r)
+		require.EqualValues(1, r.Count())
+		require.Equal(time.Hour, r.Period())
 		require.True(r.Scope(appdef.RateScope_AppPartition))
 		require.False(r.Scope(appdef.RateScope_Workspace))
 		require.False(r.Scope(appdef.RateScope_IP))

--- a/pkg/parser/types.go
+++ b/pkg/parser/types.go
@@ -574,7 +574,7 @@ type RateValueTimeUnit struct {
 }
 
 type RateValue struct {
-	Count           *int              `parser:"(@Int"`
+	Count           *uint32              `parser:"(@Int"`
 	Variable        *DefQName         `parser:"| @@) 'PER'"`
 	TimeUnitAmounts *uint32           `parser:"@Int?"`
 	TimeUnit        RateValueTimeUnit `parser:"@@"`

